Berlin/BerOrOct1270.xml

```diff
@@ -188,21 +188,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <decoDesc>
 <decoNote type="miniature" xml:id="d1">
 <locus target="#1r"></locus>
+<desc><ref type="authFile" corresp="AT1035HolyMan"></ref></desc>
+<q n="1" xml:lang="gez">ስዕለ፡ ቅዱስ፡ <gap reason="illegible"></gap>ርማም።</q>
```

Comment:
I don't think that n="1" in `<q>` is necessary when there is only one legend (or documented in the guidelines). 
I would also avoid the use of `<note>` if possible. Could the content of note be included in `<desc>` instead, or are there any conceptual objections to that?

Reply:
Thanks. I have moved the content of the note in <desc>, but only for the first miniature. I hope is fine as I did.
For  miniature 2 and 3 I left the note, the content of which refers to all their captions, more then one for each of them, in this case.